product_docs/docs/postgres_distributed_for_kubernetes/1/backup.mdx

```diff
@@ -6,31 +6,31 @@ originalFilePath: 'src/backup.md'
 EDB Postgres Distributed for Kubernetes (PG4K-PGD) supports *online/hot backup* of
 PGD clusters through physical backup and WAL archiving on an object store.
 This means that the database is always up (no downtime required) and that
-Point In Time Recovery is available.
+point-in-time recovery is available.
 
 ## Common object stores
 
-Multiple object store are supported, such as `AWS S3`, `Microsoft Azure Blob Storage`,
-`Google Cloud Storage`, `MinIO Gateway`, or any S3 compatible provider.
+Multiple object stores are supported, such as AWS S3, Microsoft Azure Blob Storage,
+Google Cloud Storage, MinIO Gateway, or any S3-compatible provider.
 Given that PG4K-PGD configures the connection with object stores by relying on
-EDB Postgres for Kubernetes (PG4K), please refer to the [PG4K Cloud provider support](https://www.enterprisedb.com/docs/postgres_for_kubernetes/latest/backup_recovery/#cloud-provider-support)
-documentation for additional depth.
+EDB Postgres for Kubernetes (PG4K), see the [PG4K cloud provider support](/postgres_for_kubernetes/latest/backup_recovery/#cloud-provider-support)
+documentation for more information.
 
 !!! Important
-    In the PG4K documentation you'll find the Cloud Provider configuration section
-    available at `spec.backup.barmanObjectStore`. Note that in PG4K-PGD examples, the object store section is found at a 
+    The PG4K documentation's Cloud Provider configuration section is
+    available at `spec.backup.barmanObjectStore`. In PG4K-PGD examples, the object store section is at a 
     different path: `spec.backup.configuration.barmanObjectStore`.
 
 ## WAL archive
 
-WAL archiving is the process that sends `WAL files` to the object storage, and it's essential to
-execute *online/hot backups*, or Point in Time recovery (PITR).
-In PG4K-PGD, each PGD Node will be set up to archive WAL files in the object store independently.
+WAL archiving is the process that sends WAL files to the object storage, and it's essential to
+execute online/hot backups or point-in-time recovery (PITR).
+In PG4K-PGD, each PGD node is set up to archive WAL files in the object store independently.
 
-The WAL archive is defined in the PGDGroup `spec.backup.configuration.barmanObjectStore` stanza,
+The WAL archive is defined in the PGD group `spec.backup.configuration.barmanObjectStore` stanza
```

Comment:
I didn't see PGDGroup (all one word defined anywhere, so I believe this is what was intended.